build_image.py

```diff
@@ -165,12 +169,6 @@ def CreateHostVMImage(RootFSDir, LinuxImage):
 def Cleanup(RootFSDir):
     shutil.rmtree(RootFSDir, ignore_errors=True)
 
-if sys.version_info[0] < 3:
-    raise Exception("Python 3 or a more recent version is required.")
-
-if (len(sys.argv) < 4):
-    sys.exit("usage: %s <Config.json> <Cache directory> <RootFS Dir>" % (sys.argv[0]))
```

Comment:
Not necessary anymore to check sys args length. parseargs takes care of that anyway.